ethcore/src/client/client.rs

```diff
@@ -899,6 +900,7 @@ impl BlockChainClient for Client {
 		}
 		let options = TransactOptions { tracing: true, vm_tracing: false, check_nonce: false };
 		let mut tx = t.clone();
+		tx.gas_price = 0.into();
```

Comment:
ok - i'll introduce an alternative way - increasing the sender account balance. slightly better than altering the tx, since we change that already in the case that there's not enough value in it.